docs/source/getting_started/ai_modules.rst

```diff
@@ -0,0 +1,49 @@
+==========
+AI Modules
+==========
+
+Overview
+--------
+
+The Citrine Platform uses artificial intelligence (AI) to accelerate your materials research and development.
+The platform exposes several types of Modules, re-usable assets that codify domain knowledge, research capabilities, or experimental objectives.
+Modules are combined into Workflows that provide actionable information to help direct research and development.
+
+The Citrine python client allows you to create and manipulate Modules and Workflows.
+Workflows can be executed and the results inspected.
+This page provides the briefest of overviews; for a more thorough discussion see the :doc:`AI Engine documentation <../workflows/getting_started>`.
+
+Modules
+-------
+
+Predictors
+^^^^^^^^^^
+
+:doc:`Predictors <../workflows/predictors>` define relations between variables.
+Predictors can be machine learning models, analytic relations, or featurizers.
+Several Predictors can be combined into a graphical model that expresses your domain knowledge and predicts material properties with quantified uncertainty estimates.
+
+Design Spaces
+^^^^^^^^^^^^^
+
+:doc:`Design Spaces <../workflows/design_spaces>` define the set of materials that can be created.
```

Comment:
That's true, but "should be evaluated" runs into problems with infinite design spaces, since it's impossible to evaluate every element of the design space. I'll change it to something more general, like "set of materials that are of interest."